capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -18,7 +22,15 @@ export class WikiMediaComponent implements OnChanges, OnInit {
   loading = true;
   body: string;
   urls = new Map();
-  cityName: string;
+  langlinks = new Map();
+  languages: any = Languages as any;
```

Comment:
Not crazy about use of "any", why is this necessary? If absolutely necessary, I think unknown type would be preferred.

Reply:
Using un-kown works but if I try casting it to anything else it doesn't seem to run properly.

Reply:
Why is it necessary in the first place?

Reply:
Nevermind, looks like you fixed it.